### DISCUSSION TITLE
Backend > Search Term - Add ID column

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#2477] [Ideas]
Go to **Backend > Catalog > Search Term**. The entry page contains the grid in which the search queries are placed in rows, implicitly the first row being the smallest ID.

The grid has 6 columns that allow sorting. If you sort one of them, a problem appears. If you have made a selection, you can no longer see the selected products, they are somewhere, but you cannot return to the initial grid ordered ascending by ID unless you log out.

My opinion is that the ID column should be added to the grid to solve this issue.

**Comment by sreichel** [comment]
> but you cannot return to the initial grid ordered ascending by ID unless you log out.
> 
> My opinion is that the ID column should be added to the grid to solve this issue.

I think ID column is not needed. Problem is, that there is "no" default sort.

We have no "name" column here, so you cant return to "default" sort order ...

See `Mage_Adminhtml_Block_Catalog_Search_Grid`

```
        $this->setDefaultSort('name');
```

This should fix it ....

```
        $this->setDefaultSort('search_query');
```

PR is ready (#2480)

**Comment by sreichel** [comment]
@ADDISON74 this should be an issue, not a discussion. The issue could have been closed, but discussion is still open.

**Reply by addison74** [reply]
If it is an issue you can convert by clicking on **Creating issue from discussion** link.

**Reply by sreichel** [reply]
I know, but i would prefer it the other way round ... if an issue is discussable, turn it into a discussion. (maybe #2341)

But an issue can be closed via PR, discussion stay open (until I hit __Delete Discussion__)

**Reply by sreichel** [reply]
Maybe we should make better issue templates ...

**Reply by addison74** [reply]
Thank you for your observation. My opinion is the following:

**Issue** = Any problem that requires detailed checks. If it is confirmed then a PR is needed. In general, malfunctions, wrong implementations, security, etc. Here we can set priorities for any of them based on labels (not implemented).

**Idea** = Any problem of functionality, improvement and many others, which require debates at the end of which they can be implemented. Also, if someone wants to implement a PR and wants feedback, they can do it in this section as well. I have many posts in this regard.. In different sections of Discussions, I have moved quite a few reports that are either questions or ideas.

In the case of the idea reported by me here, it is really a functionality issue that should have been reported in the Issue section. Similarly, the discussion about deleting composer.lock from the repository is clearly a discussion after which a conclusion must be reached, maybe even a vote for a final decision.

PS - The GitHub feature to convert a discussion to an issue takes the first post and moves it to the Issue section. I guess all further discussions are lost (needs a test).

**Comment by sreichel** [comment]
Fixed in #2480
